components/board/genericlinux/board.go

```diff
@@ -477,19 +505,25 @@ func (b *sysfsBoard) GPIOPinByName(pinName string) (board.GPIOPin, error) {
 	return nil, errors.Errorf("cannot find GPIO for unknown pin: %s", pinName)
 }
 
-func (b *sysfsBoard) Status(ctx context.Context, extra map[string]interface{}) (*commonpb.BoardStatus, error) {
+// Status returns the current status of the board.
+func (b *SysfsBoard) Status(ctx context.Context, extra map[string]interface{}) (*commonpb.BoardStatus, error) {
 	return &commonpb.BoardStatus{}, nil
 }
 
-func (b *sysfsBoard) ModelAttributes() board.ModelAttributes {
+// ModelAttributes returns attributes related to the model of this board.
+func (b *SysfsBoard) ModelAttributes() board.ModelAttributes {
 	return board.ModelAttributes{}
 }
 
-func (b *sysfsBoard) SetPowerMode(ctx context.Context, mode pb.PowerMode, duration *time.Duration) error {
+// SetPowerMode sets the board to the given power mode. If provided,
+// the board will exit the given power mode after the specified
+// duration.
+func (b *SysfsBoard) SetPowerMode(ctx context.Context, mode pb.PowerMode, duration *time.Duration) error {
```

Comment:
It's weird to see a multi-line comment with such a short width (it's clearly been line-wrapped), followed immediately by a line with a very long width (it's clearly not been line-wrapped). This is why I wish we explicitly picked a maximum line width, so we'd know what it was supposed to be. Given that you wrote the comment, that's probably the width you prefer. So, I'd break up line 521 to look like that, too. Consistency within a single paragraph or function is the most important!